backend/fms_core/resources/sample.py

```diff
@@ -177,6 +180,12 @@ def import_obj(self, obj, data, dry_run):
             individual = None
             errors["individual"] = ValidationError(e.messages.pop(), code="invalid")
 
+        if data["Project"]:
+            try:
+                self.project = Project.objects.get(name=get_normalized_str(data, "Project"))
+            except Project.DoesNotExist:
+                errors["project "] = ValidationError("Project does not exist.", code="invalid")
```

Comment:
Extra space in errors["project "]

Reply:
It could also help the user to have a more precise error such as "Project named {name} does not exist."